bitnami/nginx/templates/ingress.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+  {{- range $key, $value := .Values.ingress.labels }}
```

Comment:
Could you switch the parameter from `.Values.ingress.labels` to `.Values.ingress.selectors`?
Please also update the **values.yaml** including the new parameter and update the list of parameters in the README.md file.
Finally, you'll have to bump the minor version in the **Chart.yaml**.

Reply:
HI Juan.